File: DSA/VaniAgrawal_590015846/Day_10_Question_1.py
# Compare two strings for exact equality, case-insensitive equality, and lexicographical order. 
string1=input("string1:")
string2=input("string2:")
if string1==string2:
    output1="Equal"
else :
    output1="Not equal"

if string1.lower()==string2.lower():
    output2="Equal ignoring case"
else:
    output2=" Not equal ignoring case"

list=[string1,string2]
list.sort()
if string1 != string2:
    output3=f'"{list[0]}" comes before "{list[1]}"'
else:
    output3="Strings are identical"

print(f"{output1},{output2},{output3}")

File: DSA/VaniJain_590012020/Day10Question1.java
import java.util.*;
class Day10Question1 
{
    public static void main(String[] args) 
    {
        Scanner scr = new Scanner(System.in);
        System.out.print("String1: ");
        String str = scr.nextLine();
        System.out.print("String2: ");
        String s = scr.nextLine();
        if (str.equals(s))
            System.out.println("Equal");
        else
            System.out.println("Not Equal");
        if (str.equalsIgnoreCase(s))
            System.out.println("Equal ignoring case");
        else
            System.out.println("Not equal ignoring case");
        int compareResult = str.compareTo(s);
        if (compareResult < 0)
            System.out.println(str + " comes before " + s);
        else if (compareResult > 0)
            System.out.println(s + " comes before " + str);
        else
            System.out.println("Strings are identical");
    }
}
